Makefile

```diff
@@ -60,10 +66,26 @@ vet: ## Verifies `go vet` passes
 	@go vet $(shell go list ./... | grep -v vendor) | grep -v '.pb.go:' | grep -v '_bindata.go' | tee /dev/stderr
 
 .PHONY: install
-install: ## Installs the executable or package
-	@echo "+ $@"
+## Installs the executable or package
+install: install-bin create-declarations-dir install-man
+
+.PHONY: install-bin
+install-bin:
+	@echo "Installing binary"
 	@go install .
 
+.PHONY: create-declarations-dir
+create-declarations-dir:
+	@echo "Creating declarations dir at ~/.karn/declarations"
+	@mkdir -p ${INSTALLDECLARATIONS}
+	@cp -r ${DECLARATIONDIR} ${INSTALLDECLARATIONS}
+
+.PHONY: install-man
+install-man:
+	@echo "installing man page"
```

Comment:
Can you capitalize the 'i' for consistency?

Reply:
of course un momento porfa

Reply:
the 'i' in the echo statement i assume?